contrib/librdkafka-cmake/config.h.in

```diff
@@ -1,21 +1,29 @@
 // Originally generated by ./configure
 #ifndef _CONFIG_H_
 #define _CONFIG_H_
-#define BUILT_WITH "GCC GXX PKGCONFIG OSXLD LIBDL PLUGINS ZLIB SSL SASL_CYRUS ZSTD HDRHISTOGRAM LZ4_EXT SNAPPY SOCKEM SASL_SCRAM CRC32C_HW"
+// BUILT_WITH
```

Comment:
I think this file is okay (only minor thing is the version), but can I ask you to put a similar comment at the beginning of the file that I tried to do [here](https://github.com/ClickHouse/ClickHouse/pull/69558/files)? I think most of my comments are still true, so it might be a good base (if not the complete solution).

I hope that this will help in the future when we have to upgrade librdkafka.

Reply:
I've added your changes (slightly adjusted them)